jwst/associations/tests/test_wfss.py

```diff
@@ -25,7 +25,7 @@ def test_niriss_wfss():
     asns = results.associations
 
     # Need 4 associations: image2, spec2, image3, spec3
-    assert len(asns) == 12
+    assert len(asns) == 5
```

Comment:
As Robert mentioned in tag up this morning, this is confusing - have you tracked down why there are fewer associations after this change?

Also, if there are no regression tests that currently contain extcal data, can you please add one?

Reply:
I had misplaced the constraint and now it seems correct to me. I will add a reg test for extcal.